docs/backend/README.md

```diff
@@ -1,3 +1,13 @@
+## Setup Auth0:
+
+you will need to have a specific `.env` file to read in `AUTH0` variables (place this file in `src/py/covidr`), ask project leads for this file, there will be different `.env` files for local development, staging, and production. 
+
+if deploying to a custom staging environment change this variable in the .env file:
+
+`AUTH0_CALLBACK_URL="http://<custom-staging-env-name>.us-west-2.elasticbeanstalk.com/callback"`
+
+if you are new to working on covidr ask a project lead to ask your eb staging env to the list of allowed callbacks in Auth0. 
+
```

Comment:
I created a couple follow-on stories:

https://app.clubhouse.io/genepi/story/68840/fold-authconfig-into-main-config-object
https://app.clubhouse.io/genepi/story/68841/figure-out-how-to-set-env-variables-as-part-of-eb-deploy